src/mudlet-lua/lua/geyser/GeyserAdjustableContainer.lua

```diff
@@ -1,28 +1,29 @@
 --Adjustable Container
---Just use it like a normal Geyser Container with some extras like: 
+--Just use it like a normal Geyser Container with some extras like:
 --moveable, adjustable size, attach to borders, minimizeable, save/load...
 --right click on top border for menu
 --Inspired heavily by Adjustable Label (by Jor'Mox ) and EMCO (by demonnic )
 --by Edru 2020
 
 Adjustable = Adjustable or {}
 
---- Adjustable Container
--- @class table
--- @name Adjustable.Container
+------------
+-- Adjustable Container
+-- (LDoc)
+-- @module Adjustable.Container
 Adjustable.Container = Adjustable.Container or Geyser.Container:new({name = "AdjustableContainerClass"})
 
 local adjustInfo = {}
 
---- Internal function to add "%" to a value and round it
+-- Internal function to add "%" to a value and round it
```

Comment:
As it's a `local` function it'll never show up in the players Mudlet, so it shouldn't be in the docs.